dagql/idtui/frontend.go

```diff
@@ -470,6 +482,9 @@ func (fe *Frontend) renderProgress(out *termenv.Output) (bool, error) {
 func (fe *Frontend) renderZoomed(out *termenv.Output, st *zoomState) (bool, error) {
 	var renderedAny bool
 	for i := 0; i < st.Output.UsedHeight(); i++ {
+		if i > 0 {
+			fmt.Fprintln(out)
```

Comment:
`(*midterm.Terminal).RenderLine` no longer prints a linebreak on its own, since you wouldn't want the last line to do that.